test/membrane_opus/encoder_test.exs

```diff
@@ -19,6 +48,13 @@ defmodule Membrane.Opus.Encoder.EncoderTest do
       source: %Membrane.File.Source{
         location: @input_path
       },
+      caps_provider: %CapsProvider{
```

Comment:
Not sure what pluging CapsProvider in gives us :thinking: Does it test anything?

Reply:
Well, that's because we do not have raw audio parser and the encoder needs to receive caps somehow. An alternative would be to accept RemoteStream in encoder's input pad specification, but I didn't wanto to change the logic of the element itself.